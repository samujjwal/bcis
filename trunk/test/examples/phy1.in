unit second
unit minute = 60 * second
unit hour = 60 * minute
unit day = 24 * hour
unit year = 365 * day

set x = 1 * year

unit meter
unit kilogram
unit newton = meter * kilogram / second ^ 2

set Pi = 3.1415926
set omiga = 2 * Pi /(1*x)
print(omiga)

set G = 6.67*10 *newton * (1*meter ^ 2)/(1*kilogram^2)
print(G)

set r = 1.50 * 10^11 * meter

set m = (1*r^3) * (1*omiga^2) /(1*G)
print(m)

