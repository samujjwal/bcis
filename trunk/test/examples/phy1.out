+2E-7/second
+66.7000000*meter^3/(kilogram*second^2)
+2023988005997001197.0154391*kilogram*second^4

